bot/src/main/java/com/intern/bot/config/TelegramBot.java

```diff
@@ -15,13 +18,23 @@
 
   private final BotProperties botProperties;
   private final KafkaTemplate<String, Object> kafkaTemplate;
+  private final ObjectMapper objectMapper;
 
   @Override
   public void onUpdateReceived(Update update) {
     // We check if the update has a message and the message has text
     if (update.hasMessage() && update.getMessage().hasText()) {
-      String inputText = update.getMessage().getText();
-      kafkaTemplate.send("topic2", inputText);
+      Message inputMessage = update.getMessage();
+      kafkaTemplate.send("topic2", writeValueAsString(inputMessage));
+    }
+  }
+
+  private String writeValueAsString(Message message) {
+    try {
+      return objectMapper.writeValueAsString(message);
+    } catch (JsonProcessingException e) {
+      e.printStackTrace();
```

Comment:
This is not how logs should be printed. By default it will print into stderr, however we have `@Slf4j` annotation which creates a slf4j wrapper over some existing logs provider (in our case its log4j probably, look into dependencies). It is a good practice to rely on one logger, which decides where to print logs. https://stackify.com/slf4j-java/

Also spring will log every uncaught exception which is thrown outside of controller method. So you don't need to log it here. Your `RuntimeException` will do it for you.